config/config.go

```diff
@@ -27,11 +27,6 @@ const (
 	// DefaultLogLevel defines a default log level as INFO.
 	DefaultLogLevel = "info"
 
-	// Mempool versions. V1 is prioritized mempool, v0 is regular mempool.
-	// Default is v0.
-	MempoolV0 = "v0"
```

Comment:
I feel like we should leave some of this information in, to keep the memory of previous versions easily seen in the code so that if someone comes up with a new mempool, it does not end up as V1 again. I am not sure what would be the best approach, though. Maybe just comment out v1 and add a deprecation note in the comment as well, and leave the Version setting in, with the default value only, so people looking into the mempool will necessarily bump into the history.

Reply:
Thanks for the comment! I thought the same and I discussed it with @sergio-mena. The conclusion was that we should keep just one default implementation (the current one) and roll back the code as it was before the priority mempool was added. The reasons:
- in the future different mempools will be implemented at the application level through ABCI 1.0,
- we don't know exactly how we will structure the mempool module once we modularize Comet,
- we don't want to maintain an unused configuration entry.

Also, I think we may want to have only one default mempool implementation in Comet, and leave to the users the option to implement their own versions at the application level.

Now, I think you're right that we should document somehow that versions v0 and v1 existed in past releases (though I don't think it will be a real problem if version numbers are reused).

Reply:
I do agree, and would add that `v[0-9]` is not really a good name for different implementations.

Reply:
Cool. A good comment near MempoolConfig should suffice.

Reply:
I added a comment about the mempool versions to `MempoolConfig` in `config/config.go`.